<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Converter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="inputGroupBox">
      <property name="title">
       <string>Input File</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="inputFormatLabel">
         <property name="text">
          <string>Format:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="inputFormatCombo"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="inputFileLabel">
         <property name="text">
          <string>File:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="inputFileEdit"/>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="browseInputButton">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="outputGroupBox">
      <property name="title">
       <string>Output</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="outputFormatLabel">
         <property name="text">
          <string>Format:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="outputFormatCombo"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="outputFolderLabel">
         <property name="text">
          <string>Folder:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="outputFolderEdit"/>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="browseOutputFolderButton">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="outputFileLabel">
         <property name="text">
          <string>File Name:</string>
         </property>
        </widget>
       </item>
       <!-- Replace this in your UI file -->
       <item row="2" column="1" colspan="2">
         <widget class="QLineEdit" name="outputFileNameEdit">
           <property name="placeholderText">
             <string>Output filename will appear here</string>
           </property>
         </widget>
       </item>
       <item>
        <widget class="QLabel" name="dependencyWarningLabel">
         <property name="styleSheet">
          <string notr="true">color: red;</string>
         </property>
         <property name="text">
          <!-- <string>Note: For image conversion, ImageMagick must be installed</string> -->
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="convertButton">
      <property name="text">
       <string>Convert</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionInstallDependencies"/>
   </widget>
   <widget class="QMenu" name="menuTheme">
    <property name="title">
     <string>Theme</string>
    </property>
    <addaction name="actionLightMode"/>
    <addaction name="actionDarkMode"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTheme"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionInstallDependencies">
   <property name="text">
    <string>Install Dependencies</string>
   </property>
  </action>
  <action name="actionLightMode">
   <property name="text">
    <string>Light Mode</string>
   </property>
  </action>
  <action name="actionDarkMode">
   <property name="text">
    <string>Dark Mode</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>